hopr/hopr-lib/src/lib.rs

```diff
@@ -756,18 +789,25 @@ impl Hopr {
         // TODO: wait here for the confirmation that the node is allowed in the registry
 
         info!("Loading initial peers from the storage");
-        let index_updater = core_transport::IndexerActions::new(to_process_tx.clone());
-
         for (peer, _address, multiaddresses) in self.transport_api.get_public_nodes().await? {
             if self.is_allowed_to_access_network(&peer).await? {
                 debug!("Using initial public node '{peer}' with '{:?}'", multiaddresses);
-                index_updater
-                    .emit_indexer_update(IndexerToProcess::EligibilityUpdate(peer, PeerEligibility::Eligible))
-                    .await;
+                if let Err(e) = to_process_tx
```

Comment:
I find these chained if let statements fairly hard to read. Wouldn't it be nicer to have this be captured in a variable before and then match only the result here?